capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -81,10 +81,12 @@ class ClubPage extends Component {
       alert('Assignment creation failed. You do not own this club.');
       return;
     }
+    const dueDate = document.getElementById("due-date").value;
```

Comment:
nit: use single quotations for string literals

Reply:
good catch!

Reply:
I realized the mistake in a later PR and fixed it, but might as well fix it here. Thanks for pointing this out.